venues/ICLR.cc/2019/Conference/webfield/areachairWebfieldReviewerAssignmentStage.js

```diff
@@ -194,7 +212,7 @@ var getUserProfiles = function(userIds) {
     _.forEach(result.profiles, function(profile) {
       var name = _.find(profile.content.names, ['preferred', true]) || _.first(profile.content.names);
       profile.name = _.isEmpty(name) ? view.prettyId(profile.id) : name.first + ' ' + name.last;
-      profile.email = profile.content.preferredEmail;
+      profile.email = profile.content.preferredEmail || profile.content.emails[0];
```

Comment:
I think this change was in the first file and not here?

Reply:
Yeah, it looks like it.